examples/selfhosted/keys.h

```diff
@@ -41,230 +74,229 @@ uint8_t g_ecc_public_key_c25519[] =
-0xf2,0xa3,0x6e,0x4c,0x4a,0xd1,0x98,0xcc,0xf9,0x76,0x7f,0x8a,0x3e,0xa5,0x60,0xeb,
-0x55,0x4d,0xa5,0x36,0x1d,0x24,0x8a,0x9b,0x3f,0x3e,0xfb,0xe9,0xd7,0xdb,0x29,0xfb,
-0x9b,0x02,0x03,0x01,0x00,0x01,
+0x00,0xa2,0xcd,0x53,0x24,0x7b,0x38,0x28,0x29,0x0f,0x25,0x20,0x9b,0xc6,0xa3,0x0b,
+0xd8,0x99,0x97,0xfb,0x81,0x76,0x31,0x3f,0x13,0x9f,0x90,0x82,0xca,0x72,0x56,0xf8,
+0xf5,0x50,0x41,0x4f,0x69,0x46,0x4a,0xb6,0xb3,0x2b,0xdc,0x14,0x8b,0x13,0xd0,0xec,
+0x40,0x83,0x52,0x08,0x72,0x15,0x75,0xd7,0x79,0xd0,0xde,0x82,0xff,0xea,0xc6,0xeb,
+0xcc,0x7c,0xcf,0xb8,0xaa,0x21,0x13,0xbf,0x6c,0xc7,0xa5,0xae,0x04,0x22,0x96,0xb9,
+0x4e,0x83,0xab,0xc0,0xb4,0xa6,0x80,0x4c,0xc3,0x50,0x7a,0x2e,0x6c,0x01,0xfc,0x36,
+0xd9,0xd8,0x9b,0xc1,0xfe,0x13,0xf5,0x16,0xe1,0x57,0xad,0x6d,0xcc,0xb5,0xdd,0xf2,
+0x9d,0xbd,0x3f,0x45,0xb6,0x9f,0xf5,0x05,0x1c,0x2c,0x8f,0x4c,0xd3,0xe6,0x94,0x0c,
+0x45,0x89,0x9a,0x31,0x08,0x4f,0x47,0xc5,0x75,0x17,0xe6,0xfd,0x3c,0x26,0x51,0x06,
+0x23,0x97,0x48,0xaf,0x80,0x70,0xd8,0xe5,0xab,0x19,0x42,0x34,0x96,0x5d,0x2a,0x2a,
+0x5c,0x5d,0x21,0x10,0x17,0xeb,0xef,0x70,0xa3,0x71,0x74,0xd9,0x5a,0xe0,0x4c,0x20,
+0x84,0x9f,0x8d,0x03,0x56,0x3a,0x58,0x7c,0x8f,0x49,0xc5,0xc8,0xca,0x0f,0x45,0xf8,
+0xb4,0xd7,0x72,0x34,0x33,0xca,0xc9,0xc6,0xb7,0x19,0x16,0xaa,0xff,0x37,0x52,0x62,
+0x7a,0x34,0x30,0x55,0xd5,0x64,0xb0,0xb5,0x45,0x27,0xf2,0x54,0x02,0xe4,0xfc,0x1f,
+0x3a,0x79,0x11,0xb6,0x5d,0xb6,0x6d,0x6b,0x94,0x23,0x49,0xba,0xb1,0x6b,0x34,0x68,
+0x47,0x1a,0x32,0x71,0x83,0x5c,0x20,0x47,0x80,0x40,0xed,0xf5,0xc8,0x20,0x75,0x31,
+0x71,0x4e,0x50,0x06,0x10,0xed,0x27,0x01,0xa6,0x39,0x61,0x96,0xa4,0x1e,0x71,0xfd,
+0xbf,0x55,0x3c,0x04,0x48,0x2a,0x3c,0x70,0x80,0x29,0x76,0xf5,0xfc,0x50,0x59,0x12,
+0xa6,0x18,0x46,0xba,0xa2,0x10,0x45,0x2a,0x5e,0xc6,0xc9,0x5f,0x37,0x14,0x7a,0x35,
+0x59,0x85,0xe6,0xcb,0x28,0xbd,0xcf,0x0c,0xf1,0xb2,0xf1,0x50,0xea,0x26,0x05,0x3d,
+0x89,0x50,0x23,0x77,0x78,0x93,0xfd,0xca,0xcf,0x53,0xc3,0x34,0x59,0x73,0xad,0x72,
+0x2e,0x59,0x45,0xd5,0x04,0x5a,0x59,0x47,0x37,0x8e,0x93,0xe4,0xa2,0x48,0x3a,0xca,
+0xe7,0x7c,0x5c,0x98,0xa7,0xda,0x8e,0x88,0x03,0xda,0xff,0xd9,0x82,0x17,0xf3,0x76,
+0x2b,0xb0,0x6b,0x47,0x1d,0xf3,0x46,0x99,0x6b,0x7c,0xea,0xd9,0xce,0xc7,0x1d,0x87,
+0x22,0x08,0x9c,0x21,0xd8,0x2a,0x1e,0x8b,0x39,0x13,0x43,0xa2,0x8c,0xb4,0xf8,0x68,
+0xc0,0xa3,0xbb,0x34,0xc7,0xf7,0xaf,0xea,0xbf,0x5e,0xc0,0x24,0xc3,0x8a,0x2e,0x53,
+0x11,0x7a,0x71,0x1e,0x58,0x0f,0xd6,0xfe,0x9b,0xd1,0xc3,0xec,0x8a,0x0a,0xb6,0x90,
+0xda,0x8d,0x18,0x96,0xf6,0x28,0xe7,0xbb,0xf1,0x36,0xec,0xb6,0xe3,0x48,0x03,0x8d,
+0x49,0xf0,0xe2,0xac,0x89,0x5e,0x02,0x36,0x6a,0xf2,0xba,0xcd,0x31,0x33,0xe3,0x3d,
+0x48,0xe6,0xfd,0xdf,0x51,0xbe,0xa0,0x8a,0x40,0x81,0xda,0xfd,0x48,0x15,0xbd,0x43,
+0xf0,0x1e,0x95,0x1d,0x60,0x97,0x3d,0xec,0xe5,0x50,0xd7,0x82,0xd2,0x92,0x5e,0xf7,
+0x1d,0x4f,0xfb,0xaf,0xfb,0x19,0xd4,0x49,0xcc,0xf3,0xc7,0xf6,0x20,0xf3,0xbd,0x6e,
+0x61,0x02,0x03,0x01,0x00,0x01,
 };
 
 uint8_t g_dsa_message[] =
 {
-/* SHA256 of "Tempus unum hominem manet." */
```

Comment:
This changes the intended functionality, the (pre)hashing of the TLS and other messages is already included in the separate hash calls, this value should remain the hash of the message, not the message itself.